src/data/equiv/mul_add.lean

```diff
@@ -213,6 +213,18 @@ e.to_equiv.symm_apply_eq
 lemma eq_symm_apply (e : M ≃* N) {x y} : y = e.symm x ↔ e y = x :=
 e.to_equiv.eq_symm_apply
 
+lemma eq_comp_symm {α β γ} (e : α ≃* β) (f : β →* γ) (g : α →* γ) :
+  f = g ∘ e.symm ↔ f ∘ e = g := by {split; intro h; ext, simp [h], simp [←h]}
+
+lemma comp_symm_eq {α β γ} (e : α ≃* β) (f : β →* γ) (g : α →* γ) :
+  g ∘ e.symm = f ↔ g = f ∘ e := by {split; intro h; ext, simp [←h], simp [h]}
+
+lemma eq_symm_comp {α β γ} (e : α ≃* β) (f : γ →* α) (g : γ →* β) :
+  f = e.symm ∘ g ↔ e ∘ f = g := by {split; intro h; ext, simp [h], simp [←h]}
+
+lemma symm_comp_eq {α β γ} (e : α ≃* β) (f : γ →* α) (g : γ →* β) :
+  e.symm ∘ g = f ↔ g = e ∘ f := by {split; intro h; ext, simp [←h], simp [h]}
```

Comment:
These need `@[to_additive]` and should use the local `M`, `N` and `P` variables

Reply:
Oops I copy pasted them quickly and forgot that